wd_demo.au3

```diff
@@ -55,7 +56,7 @@ Func _WD_Demo()
 	Local $aCheckboxes[$iCount]
 
 	Local $hGUI = GUICreate("Webdriver Demo", 200, 150 + (20 * $iCount), 100, 200, BitXOR($GUI_SS_DEFAULT_GUI, $WS_MINIMIZEBOX))
-
+	GUISetBkColor($COLOR_LIGHTSTEELBLUE)
```

Comment:
This doesn't compile for me. Are you running beta? I don't see this constant in ColorConstants.au3

Reply:
Confirmed that $COLOR_LIGHTSTEELBLUE is available in latest beta, but we want this to work without requiring beta. Can you retest using a different color, such as $CLR_SILVER or $CLR_MEDGRAY?